### DISCUSSION TITLE
Cannot connect to S3 endpoint

### DISCUSSION THREAD

**Discussion by rewse** [vrtmrz/obsidian-livesync#418] [Q&A] [answered]
“Error! Could not connected to https://s3.dualstack.ap-northeast-1.amazonaws.com/ Failed to fetch” error happens when Test Connection is tried. I also tried https://s3.ap-northeast-1.amazonaws.com/ , but it doesn’t work either.

What should I define as the Endpoint URL for S3? I’m referring to https://docs.aws.amazon.com/general/latest/gr/s3.html for the S3 endpoint.

```
5/5/2024, 3:16:02 PM->Error! Could not connected to https://s3.dualstack.ap-northeast-1.amazonaws.com/
Failed to fetch
5/5/2024, 3:16:02 PM->TypeError:Failed to fetch
```

**Comment by rewse** [comment]
I found [the doc](https://github.com/vrtmrz/obsidian-livesync/blob/main/docs/quick_setup.md) mentioning this setting, but doesn’t replicate. Let me open an issue.

> Note 1: if you use S3, you can leave the Endpoint URL empty.

**Comment by rewse** [comment, marked as answer]
I found the correct way for S3.

Endpoint URL: _empty_
Access Key: Your access key (e.g. AIMAHHANGEE6XAIW4REV)
Secret Key: Your secret key (e.g. zau3aG9ooxoo1jah7xiocepe+shou1ru9daizeiy)
Region: Your region. **The default value, _auto_ desesn’t work for S3** (e.g. ap-northeast-1)
Bucket Name: Your bucket name (e.g. samplebucket)
Use Custom HTTP Handler: On

Tring Test Connection, you would get PM->Connected to  successfully!
If you still don’t succeed, View > Toggle Developer Tools > Console is helpful to debug.

**Reply by vrtmrz** [reply]
Thank you for your investigation! Swallow my shame, I had not tried the official S3. 
I will fix the document soon!
